app/src/main/java/com/github/fribourgsdp/radio/Song.kt

```diff
@@ -0,0 +1,38 @@
+import java.util.*
+
+class Song (songName: String, artistName: String, songLyrics: String) {
+
+    var name: String = reformat_song_name(songName)
+        get() {return field}
+        set(value) {field = reformat_song_name(value)}
+
+    var artist: String = artistName
+    var lyrics: String = songLyrics
```

Comment:
Why not make all of these as `val` to make a `Song` immutable?

Reply:
For custom playlists made by users I thought it would be nice to allow users the chance to change the song name.